indico/modules/events/util.py

```diff
@@ -501,3 +502,10 @@ def serialize_event_for_ical(event, detail_level):
     if detail_level == 'contributions':
         data['contributions'] = [serialize_contribution_for_ical(c) for c in event.contributions]
     return data
+
+
+def render_acl(obj):
```

Comment:
It's not really event related.. maybe it'd be better off in indico/core/db/sqlalchemy/protection.py since that's where the related ACL mixins etc are defined?